video_player.py

```diff
@@ -2,8 +2,13 @@
 from PyQt5.QtMultimedia import QMediaPlayer, QMediaContent
 from PyQt5.QtMultimediaWidgets import QVideoWidget
 from PyQt5.QtCore import QUrl, pyqtSignal
+import logging
+from datetime import date
 
 
+logging.basicConfig(filename=f'./assets/logs/{date.today()}.log', level=logging.DEBUG)
```

Comment:
As we discussed before, you only call `basicConfig` once from your `main()` or somewhere similar.

Reply:
✅